### ISSUE TITLE
Ensure output is turned off

### ISSUE THREAD

**Issue by sclarke81** [XRobots/WiperMotorServo#3]
I think you need an else to ensure both PWM channels are turned off if Output1 is zero. As it is when Output1 transitions to zero one of the PWM channels will be left driving (admittedly a small value because of the PID).

**Comment by XRobots** [comment]
Yes you could also leave a deadzone, although the last time I did that in code a bunch of people said it should be there at all and PID should take care of it.

**Comment by jakezilla** [comment]
I think what @sclarke81 is talking about is this section of code
```
      if (Output1 > 0) {

        analogWrite(5, Output1);

        analogWrite(6, 0);

      }

      else if (Output1 < 0) {

        Output1a = abs(Output1);

        analogWrite(5, 0);

        analogWrite(6, Output1a);

      }
```
if Output1 = 0, no analog writes happen. If you change "Output1 > 0" to "Output1 >= 0" all cases will be included.

**Comment by sclarke81** [comment]
@jakezilla Yup, exactly that.
